docs/operations/faq/overview.rst

```diff
@@ -0,0 +1,12 @@
+.. _faq_overview:
+
+Envoy FAQ
+==========
+
+The purpose of this document is to cover commonly used Envoy configuration and scenarios in typical
+Envoy deployment, e.g., how to setup zone aware routing. 
+
+.. toctree::
```

Comment:
hm, there is already separate rst, for example, for zone aware routing there is separate file.
there will be separate file for lightstep tracing etc.
let's sync offline